### PR TITLE
Added french

### PR DESCRIPTION
**Pull request by makouille**
- Added the fr/ directory in README.md
- Added the fr/ directory in locales (2 command files data + "common.json" just like the other files)
- Added the fr imports in ressources.ts

### CODE REVIEW

**Review by RomaDevWorld — CHANGES_REQUESTED** [review]
Hey, thanks for contributing to my repo.

I just made a few changes in 8ball command, so your files are a little bit outdated. Please check[ this commit ](https://github.com/RomaDevWorld/Cossack-BOT/commit/c2b4a480c5cc5b76e12dd7823ed839432e03112b) and add few more lines to `8ball.json`
Will merge as soon as you do this.

Thanks. Have a great day!

**Comment by RomaDevWorld** [comment]
Also, you seem to forgot to localize commands descriptions
Look inside one of `/src/commands`, and you will see something like this:
```
    .setDescription('Create a table for dice')
    .setDescriptionLocalizations({ uk: 'Створити стіл для гральних кубиків'  }),
```

Just add `fr` to ` .setDescriptionLocalizations` with your text

File: src/commands/purge.ts [hunk 1]
```diff
@@ -18,11 +19,13 @@ const command: SlashCommand = {
         .setDescription('Max amount of messages')
         .setDescriptionLocalizations({
           uk: 'Максимальна кількість повідомлень',
+          fr: 'Nombre maximum de messages'
         })
     )
     .addUserOption((option) =>
       option.setName('user').setDescription('Delete messages from a specific user').setDescriptionLocalizations({
         uk: 'Видалення повідомлень від вказаного користувача',
+        fr: 'Supprimer les messages d'un utilisateur spécifique'
```
Comment (RomaDevWorld) [inline src/commands/purge.ts:28]: Syntax error here?

**Comment by makouille** [comment]
- The new syntax done for the embed name/fotter on 8ball.json
- Fixed purge.ts typo
- Description localizations
Tell me what else need to be fixed :D

**Comment by RomaDevWorld** [comment]
Looks good.
No idea why doesn't it run automatic checks..
Anyway, I'll clone and run it real quick to verify that everything is working correctly.

**Comment by makouille** [comment]
Oh I had to go myself resolve those "problems" (it just added some weird characters and "main" over the edited file and told me those were problems, so I deleted them so that the file would be just like it was before and now the branch has no conflicts :D

**Comment by RomaDevWorld** [comment]
Guess that's all. Thanks again for your participating!

**Pull request merged**